packages/angular/projects/clr-angular/src/forms/range/range-container.ts

```diff
@@ -22,13 +22,20 @@ import { IfControlStateService } from '../common/if-control-state/if-control-sta
       <div class="clr-range-wrapper" [class.progress-fill]="hasProgress">
         <ng-content select="[clrRange]"></ng-content>
         <span *ngIf="hasProgress" class="fill-input" [style.width]="getRangeProgressFillWidth()"></span>
-        <clr-icon
+        <cds-icon
           *ngIf="showInvalid"
           class="clr-validate-icon"
+          status="danger"
           shape="exclamation-circle"
```

Comment:
Why isn't this module adding them like the others do?
```
export class ClrRangeModule {
  constructor() {
    ClarityIcons.addIcons(exclamationCircleIcon, checkCircleIcon);
  }
}
```

Reply:
Somehow I missed that module. Thank you for noticing it.